py3status/modules/google_calendar.py

```diff
@@ -175,6 +188,15 @@ def _get_credentials(self):
 
         Returns: Credentials, the obtained credential.
         """
+        client_secret_path = os.path.dirname(self.client_secret)
+        auth_token_path = os.path.dirname(self.auth_token)
+
+        if not os.path.exists(auth_token_path):
+            os.makedirs(auth_token_path)
```

Comment:
(Opinion) We make dirs, but do we place the files in there? If we are not doing that, then maybe we should only check and report as it might be the cleaner approach as this could be a little too much. @tobes Maybe it'd be nice for py3status to start using `~/.config/py3status` in the near future... `rate_counter` could use it too.

Reply:
`client_secret`, we don't place, no. But `auth_token` does get created after authorization and put in `.config\py3status`. I see your point, though. I can take that out if need be.